change the datastore to PostgreSQL

wbtaro opened the pull request:
change storage from file to PostgreSQL

wbtaro committed:
change the datastore to PostgreSQL

wbtaro committed:
Fix: edit files to store memos to PostgreSQL

wbtaro committed:
Delete: delete memo directory because it is not necessary

komagata commented on memo.rb:
```diff
@@ -1,58 +1,74 @@
 
 require "securerandom"
+require "pg"
+require "./db_info"
```
こういう場合はrequire_relativeを使うとよいです〜

komagata commented on memo.rb:
```diff
@@ -1,58 +1,74 @@
+    conn = PG.connect CONNECT_STRING
+    conn.exec("Select * from memo") do |result|
+      result.each do |row|
+        p row
```
ここで出力するとターミナルに不要な出力が出ちゃってるかもです。

wbtaro committed:
Fix: fix some files in response to review

chages are show as follows
  - get connect string from environment variables
  - change require to require_relative for loading "db_info.rb"
  - delete debug code "p xxx"

komagata commented on db_info.rb:
```diff
@@ -0,0 +1,12 @@
+class DbInfo
+  def self.connect_string
+    {
+      host: "13.78.57.122",
```
hostもマジックナンバーになるのでどうにかしたいですね。

wbtaro committed:
Fix: get all information from environment variables

komagata reviewed (approved):
確認しました、OKですー🙆‍♂️
